### DISCUSSION TITLE
When used with logging，new porgress bar printed instead of refreshing existing porgress bar eachtime progress updates

### DISCUSSION THREAD

**Discussion by majingjaor** [Textualize/rich#2607] [Q&A]
Hi guys,

I'm trying to use rich progress bar together with python logging. But each time progress updates, a new progress bar will be printed on my teminal.

![image](https://user-images.githubusercontent.com/35094989/197394222-3fca39ff-cf89-49f0-be14-3a686454e3d9.png)

I tried logger.addHandler() and logging.basicConfig() and the problem occured with both ways when :

1. set a handler with a RichHandler with a rich console
2. set a handler with a logging.StreamHandler()

Is there anyone knowing the reason why this happened?

Here is my code:

```python
import time
import logging
from rich.logging import RichHandler

from rich.progress import Progress
from rich.console import Console
from rich.theme import Theme

console = Console(color_system="truecolor")

logging.basicConfig(level=logging.DEBUG, handlers=[RichHandler(console=console), logging.FileHandler('aaa.log', mode='w')])
#logging.basicConfig(level=logging.DEBUG, handlers=[logging.StreamHandler(), logging.FileHandler('aaa.log', mode='w')]

logger = logging.getLogger('rich')

with Progress(expand=True) as progress:
    task = progress.add_task("Loading...",)
    while not progress.finished:
        logger.info('some log output')
        progress.update(task, advance=10)
        time.sleep(1)
```

**Comment by rbonthond** [comment]
works for me if you also assign the console to Progress, like this:

```
progress = Progress(console=console)
with progress:

```
